app/controllers/stories_controller.rb

```diff
@@ -85,6 +85,13 @@ def create
     end
   end
 
+  def history
+    @story = policy_scope(Story).try(:find, params[:story_id])
```

Comment:
The access restrictions on this action for a story are exactly the same as for the show action. So, I've just used the same. Also, I am still trying but until now, I was not able to get a nil value from it yet. Do you have any suggestions on how can I improve it?